data-serving/scripts/converters.py

```diff
@@ -0,0 +1,121 @@
+'''Library of conversion functions for converting CSV line-list data to json.'''
+
+import datetime
+import logging
+import math
+import numbers
+import pandas as pd
+from typing import Any, Callable, Dict, List
+
+
+VALID_SEXES = ['female', 'male', 'other']
+
+range_ct = 0
+single_ct = 1
+
+
+def convert_range(value: str, parse_fn: Callable[[str], str], format_fn: Callable[[Any], Any]):
+    '''
+    Converts either a string representation of a single value or a range of values into a range object. The values can be of any types using the custom parse and format functions.
```

Comment:
Nit: linebreak this?

Not sure what editor you're using, but also check out `autopep8` if you haven't added a formatter yet.

Reply:
so vscode is using autopep8 to format on save, but it doesn't seem to do comments right? it also *looks* line it's wrapping in vscode but if i less it, it clearly isn't